drafts/one-percent-for-open-source.rst

```diff
@@ -0,0 +1,103 @@
+the operating system to the cryptography,
+all of it is open source.
+
+Open source has a `funding problem`_.
+There are many `warning signs`_ that the future is not as bright as the past.
+
+Open source is maintained for "free" by people all over the world,
+enabled by the internet (which is also built on open source).
+Many people doing this work are burning out,
+giving up,
+or just never standing in the first place.
+We need to enable the sustainability of open source in order to protect this commons that we currently have,
+and help it grow into the future.
+
+Funding Open Source
+-------------------
+
+I'd like to propose an idea:
+
+    A program called **1% for open source**: A program where Venture Capitalist's and startup companies agree to invest 1% of their funding round into the open source software that they use. 
+
+I believe that this is a win/win/win for all involved:
+
+* **Venture Capitalists** invest in the open source ecosystem that makes companies *much cheaper to build*. It's also *really good marketing* for the VC firms, to show they are investing in the common tools we all rely on.
+* **Startups** invest in the tools that they use, to *ensure continued support and maintenance*, and remove risk from their future. It will also make *hiring developers* much easier for the companies, because developers love companies that invest in open source.
+* **Open source projects** are able to *fund project management and maintenance* that is required to make them sustainable for the long term. This continues the *force multiplier* that open source has given to the software industry as a whole.
+
+Building a standard model for investing in open source,
+and a point in time where we do it,
+will help create more investment in our existing infrastructure.
+New software projects will also be created,
+because now there is a more workable sustainability model.
+
+Why Venture Capital payments?
+-----------------------------
+
+Everyone involved in the above transaction gets values from open source.
+I believe that everyone also gets value from the 1% for open source program,
+through marketing and better software over time.
+
+Having a transaction that is already happening,
+with all parties benefiting,
+is a natural place to redistribute some of the money that is changing hands.
+
+How should we implement it?
+---------------------------
+
+The startup who has gotten the investment will reinvest 1% of the money from the VC firm into the open source projects that they use.
+
+There are many different ways this could be implemented as a lower level.
+A few of my ideas are:
+
+* Startup companies create a list of the OSS projects they use, using tools like http://libraries.io/, and then give money to the 10 largest projects that are accepting money
+* There could be a new non-profit foundation created, which gets all the donations, and has people on staff who manage distributing the money fairly
```

Comment:
While this is the real answer to make this work, it does sound a bit bureaucratic so I might not talk about "people on staff".